response_operations_ui/controllers/reporting_units_controllers.py

```diff
@@ -49,6 +49,26 @@ def change_enrolment_status(business_id, respondent_id, survey_id, change_flag):
                  business_id=business_id, respondent_id=respondent_id, survey_id=survey_id, change_flag=change_flag)
 
 
+def change_respondent_status(respondent_id, change_flag):
+    if change_flag == 'UNLOCKED':
+        change_flag = 'ACTIVE'
```

Comment:
Why not just give 'ACTIVE' as the value for change_flag param?

Reply:
fair play